paddle/fluid/operators/ngraph/ngraph_bridge.cc

```diff
@@ -32,23 +32,60 @@ std::map<std::string,
                                 std::string, std::shared_ptr<ngraph::Node>>>)>>
     NgraphBridge::NG_NODE_MAP = {
         {"accuracy", NG_OPS::BuildAccuracyNode},
```

Comment:
It's not elegant to list all the NG_NODE_MAP here. Could you generate it automatically?

Reply:
Sure, I've been working on this issue, yet.